IntegrationTests/UITests/CommandsDialogs/FormBrowseTests.LeftPanel.ReorderNodes.cs

```diff
@@ -174,12 +188,12 @@ private void ValidateOrder(List<TreeNode> initialNodes, TreeNodeCollection currN
             }
         }
 
-        private void RunFormTest(Action<FormBrowse> testDriver)
+        private void RunFormTest(Action<RepoObjectsTree.TestAccessor> testDriver)
         {
             RunFormTest(
                 form =>
                 {
-                    testDriver(form);
+                    testDriver(form.GetTestAccessor().RepoObjectsTree.GetTestAccessor());
```

Comment:
❕ It's fine to make the change to pass `RepoObjectsTree` here (with the comment about renaming the method applying), but we should not be passing the `TestAccessor` directly.

```suggestion
                    testDriver(form.GetTestAccessor().RepoObjectsTree);
```